modules/default/currentweather/currentweather.js

```diff
@@ -135,7 +135,7 @@ Module.register("currentweather", {
 			var windDirection = document.createElement("sup");
 			if (this.config.showWindDirectionAsArrow) {
 				if (this.windDeg !== null) {
-					windDirection.innerHTML = ' &nbsp;<i class="fa fa-long-arrow-down" style="transform:rotate(' + this.windDeg + 'deg);"></i>&nbsp;';
+					windDirection.innerHTML = ' &nbsp;<i class="fas fa-long-arrow-down" style="transform:rotate(' + this.windDeg + 'deg);"></i>&nbsp;';
```

Comment:
```suggestion
					windDirection.innerHTML = ' &nbsp;<i class="fas fa-long-arrow-alt-down" style="transform:rotate(' + this.windDeg + 'deg);"></i>&nbsp;';
```
<img width="180" alt="Bildschirmfoto 2022-01-05 um 13 41 12" src="https://user-images.githubusercontent.com/9592452/148219259-c6a3a7c4-e7ce-43f0-9353-ab2f73e88ceb.png">
<img width="146" alt="Bildschirmfoto 2022-01-05 um 13 41 04" src="https://user-images.githubusercontent.com/9592452/148219256-807f12de-2ede-43f5-91a2-006c956e34e6.png">